### DISCUSSION TITLE
Un meme...

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-IdSw1#125] [General]
https://x.com/5eniorDeveloper/status/1856156950519853309

Hay respuestas que dan miedo...

![image](https://github.com/user-attachments/assets/e8611630-cf68-48f9-8e31-dba915511545)

**Comment by TheMoys** [comment]
Una forma fácil de recordar

![WhatsApp Image 2024-11-20 at 2 38 48 PM](https://github.com/user-attachments/assets/97f2447b-e554-4556-8920-9a34446c845b)

**Reply by mmasias** [reply]
Genial! Lo he incluido en el [artículo donde vemos estos nombres](https://github.com/mmasias/PRG1/blob/main/documentos/nomenclaturas.md)
